kubeflow/new-package-stub/parts.yaml

```diff
@@ -1,22 +1,16 @@
-{
-   "name": "give-this-package-a-name",
-   "apiVersion": "0.0.1",
-   "kind": "ksonnet.io/parts",
-   "description": "An empty package used as a stub for new packages.\n",
-   "author": "kubeflow-team <kubeflow-discuss@googlegroups.com>",
-   "contributors": [
-   ],
-   "repository": {
-      "type": "git",
-      "url": "https://github.com/kubeflow/kubeflow"
-   },
-   "bugs": {
-      "url": "https://github.com/kubeflow/kubeflow/issues"
-   },
-   "keywords": [
-      "kubernetes",
-      "kubeflow",
-      "machine learning"
-   ],
-   "license": "Apache 2.0",
-}
+name: "${PACKAGE_NAME}"
```

Comment:
interesting... parts.yaml is a json.
Do you know what this file is for?
I randomly checked other packages, and their parts.yaml are json and are fine?

Reply:
This is the file that is copied and pasted for creating new components. Everyone else has parts.yaml because they copied this file. I found it confusing to be json when the file is clearly named .yaml.

Reply:
Additionally, i tried to make it more clear which parts needed to substituted by the user.

Reply:
https://github.com/ksonnet/parts/blob/master/incubator/memcached/parts.yaml

The official ksonnet examples are also json though...

Reply:
Asking them: https://github.com/ksonnet/parts/issues/71

Reply:
Agree that changing this to yaml is less confusing